### PR TITLE
typo: not nVidia nor NVidia, but NVIDIA

### PR DESCRIPTION
**Pull request by tomoaki0705**
* use correct name for NVIDIA
  * remove "NVidia" and "Nvidia"
  * keep the letters for API

### CODE REVIEW

File: cmake/templates/cvconfig.h.in [hunk 1]
```diff
@@ -49,13 +49,13 @@
 /* C= */
 #cmakedefine HAVE_CSTRIPES
 
-/* NVidia Cuda Basic Linear Algebra Subprograms (BLAS) API*/
+/* NVIDIA Cuda Basic Linear Algebra Subprograms (BLAS) API*/
```
Comment (alalek) [inline cmake/templates/cvconfig.h.in]: Cuda => CUDA ?

Comment (tomoaki0705) [reply cmake/templates/cvconfig.h.in]: Nice review !
Cuda -> CUDA pushed.

**Comment by tomoaki0705** [comment]
Some reference based on Google search
http://international.download.nvidia.com/partnerforce-us/Brand-Guidelines/NVIDIA_Corporate_Guidelines_2012_1.2.pdf
https://twitter.com/_ksasaki/status/862203096671203332 (Tweet in Japanese, but this person is from NVIDIA)
https://linustechtips.com/main/topic/230283-how-to-spell-nvidia/

**Comment by tomoaki0705** [comment]
Thank you ! updated.

**Pull request merged**